packages/@expo/cli/src/start/server/middleware/ExpoGoManifestHandlerMiddleware.ts

```diff
@@ -103,7 +103,8 @@ export class ExpoGoManifestHandlerMiddleware extends ManifestMiddleware<ExpoGoMa
       requestOptions
     );
 
-    const runtimeVersion = Updates.getRuntimeVersion(
+    const runtimeVersion = await Updates.getRuntimeVersionAsync(
+      exp._internal?.projectRoot,
```

Comment:
```suggestion
      this.projectRoot,
```

middleware has the derived projectRoot already. we don't have to use the internal property.